sql-cli/conftest.py

```diff
@@ -115,7 +119,7 @@ def sample_dag():
     yield DAG(dag_id, start_date=DEFAULT_DATE)
     with create_session() as session_:
         session_.query(DagRun).delete()
-        session_.query(TI).delete()
+        session_.query(Ti).delete()
```

Comment:
Why this change?

Reply:
Pycharm complains that camelCase imported as constant, so had made that change. But, reverted now.

<img width="1006" alt="Screenshot 2022-11-17 at 12 56 22 PM" src="https://user-images.githubusercontent.com/10206082/202382724-99b6a092-85e7-41a5-b467-b54e51fc789c.png">